frontend/src/components/DevTools.jsx

```diff
@@ -0,0 +1,271 @@
+import React, { useEffect, useState } from 'react';
+import Navbar from './Navbar';
+import '../styles/DevTools.css';
+
+const fillerWords = ["is", "are", "now", "the", "and", "a", "an", "in", "on", "to", "from", "for", "with", "of", "at", "by", "but", "or", "nor", "so", "too", "very", "can", "could", "will", "would", "should", "shall", "may", "might", "must", "ought"];
+
+const DevTools = () => {
+    const [overwatchData, setOverwatchData] = useState([]);
+    const [leagueData, setLeagueData] = useState([]);
+    const [error, setError] = useState('');
+    const [words, setWords] = useState([]);
+    const [keywords, setKeywords] = useState([]);
+    const [classifiers, setClassifiers] = useState([]);
+    const [nerfs, setNerfs] = useState([]);
+    const [buffs, setBuffs] = useState([]);
+    const [selectingFor, setSelectingFor] = useState(null);
+    const [selectedKeyword, setSelectedKeyword] = useState(null);
+
+    useEffect(() => {
+        const fetchAllData = async () => {
+            try {
+                const [overwatchResponse, leagueResponse] = await Promise.all([
+                    fetch('http://localhost:3000/patchnotes/overwatch/'),
+                    fetch('http://localhost:3000/patchnotes/league-of-legends/')
+                ]);
+
+                if (!overwatchResponse.ok || !leagueResponse.ok) {
+                    throw new Error('Network response was not ok');
+                }
```

Comment:
if one fails both fail?

Reply:
yeah let's split out the error messages to be more informative about which game failed

Reply:
Also just thinking that if one fails the other should still render (unclear if this is a blocking exception)